docs/api-reference/app.md

```diff
@@ -10,22 +10,19 @@ An Aragon app installed in an organization from an [aragonPM repository](https:/
 | `address` | `String` | The address of the app proxy contract \(never changes\). |
 | `appId` | `String` | The `appName` encoded via [namehash](https://eips.ethereum.org/EIPS/eip-137). |
 | `appName` | `String` | The app's ENS identifier. E.g. `"token-manager.aragonpm.eth"` |
```

Comment:
A few of the properties now are `get` methods. Should we keep them as properties or move them as methods?

Reply:
I think it’s better to keep them as property, since that’s how they feel from the outside.